baseband/mark4/tests/test_mark4.py

```diff
@@ -751,3 +752,68 @@ def test_file_streamer(self, tmpdir):
             orig_bytes = fr.read(number_of_bytes)
             conv_bytes = fh.read()
             assert conv_bytes == orig_bytes
+
+
+class Test16TrackFanout4():
+    def test_find_frame(self):
+        with mark4.open(SAMPLE_16TRACK, 'rb') as fh:
+            assert fh.find_frame(ntrack=16) == 22124
+
+    def test_header(self):
+        with open(SAMPLE_16TRACK, 'rb') as fh:
+            fh.seek(22124)
+            header = mark4.Mark4Header.fromfile(fh, ntrack=16, decade=2010)
+
+        # Try initialising with properties instead of keywords.
+        # * time imply the decade, bcd_unit_year, bcd_day, bcd_hour,
+        #   bcd_minute, bcd_second, bcd_fraction;
+        # * ntrack, samples_per_frame, bps define headstack_id, bcd_track_id,
+        #   fan_out, and magnitude_bit;
+        # * header.converter since lsb_output and converter_id are somewhat
```

Comment:
A quick check shows that you can just omit those -- `fromvalues` sets `nsb=1` by default, and that does the right thing. I'd pass in  `nsb=1` to be more explicit, though.